server.js

```diff
@@ -246,20 +246,23 @@ async function getNetwork () {
 	return network;
 }
 
-function sendTransaction (abi, address, method, args) {
+async function sendTransaction (abi, address, method, args) {
 	let o = api.newContract(abi, address);
 	let tx = {
 		from: account.address,
 		to: address,
-		gasPrice: account.gasPrice,
 		data: o.getCallData(o.instance[method], {}, args)
 	};
+	if (account.gasPrice) {
+		tx.gasPrice = account.gasPrice;
+	}
 	console.log('Sending transaction: ', tx);
 
-	const hash = account.password === null
+	const request = account.password === null
 		? api.eth.sendTransaction(tx)
 		: api.personal.signAndSendTransaction(tx, account.password);
 
+	const hash = await request;
```

Comment:
Why not have the `await` for each api calls ?

Reply:
Good point. Fixed.